dashboard/db/migrate/20200403195509_create_lesson_groups.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLessonGroups < ActiveRecord::Migration[5.0]
+  def change
+    create_table :lesson_groups do |t|
+      t.string :name
+      t.integer :unit_id
```

Comment:
this column should be not null, because a lesson group should always belong to a script. we can relax this later if we find that we need to. the factory may need to be updated to auto-create a script, the same way that the stage factory does today: https://github.com/code-dot-org/code-dot-org/blob/2f2c424912918a9278b03410aecd88edb28ad967/dashboard/test/factories/factories.rb#L728-L730